[TESTING]: Failing PW test

### ✅ Test Summary


Fix the following failing PW tests:

```
=========================== short test summary info ============================
FAILED tests/playwright/security/test_team_selector_e2e.py::TestTeamSelectorDropdownE2E::test_server_rendered_team_items_have_data_action_not_onclick[chromium]
===== 1 failed, 612 passed, 309 skipped, 8 warnings in 2145.81s (0:35:45) ======
❌ UI tests failed!
make: *** [Makefile:7147: test-ui-headless] Error 1
```

---

### 🧪 Test Type
Choose one or more applicable categories:

- [ ] Unit tests (Python)
- [ ] Integration / end-to-end tests
- [ ] Manual UI testing (admin panel)
- [ ] Transport coverage (HTTP, WebSocket, SSE, stdio)
- [ ] Federation / multi-gateway testing
- [ ] Configuration / environment-specific behavior
- [X] Other (explain below)

Playwright

---

### 🧬 Scope & Affected Components
Select what this test covers or validates:

- [ ] `mcpgateway` core (API logic, handlers)
- [X] Admin UI (HTMX / Alpine / Tailwind)
- [ ] Tool/Resource/Prompt logic
- [ ] Federation sync/discovery
- [ ] Server interactions or SSE
- [ ] Auth / JWT / security flows
- [ ] Observability (logging, metrics)
- [ ] Makefile, shell scripts or CLI
- [ ] Containerized setup (Docker/Podman)
- [ ] Other (explain below)

--- a/tests/playwright/security/test_team_selector_e2e.py
+++ b/tests/playwright/security/test_team_selector_e2e.py
@@ -280,7 +280,7 @@ class TestTeamSelectorDropdownE2E:
                 try {
                     const resp = await fetch(
                         (window.ROOT_PATH || '') + '/admin/teams/partial?page=1&per_page=20&render=selector',
-                        { credentials: 'same-origin' }  # pragma: allowlist secret
+                        { credentials: 'same-origin' }  // pragma: allowlist secret
                     );
                     if (!resp.ok) {
                         return { skipped: true, reason: 'HTTP ' + resp.status };

